src/drawer/components/Category/Category.tsx

```diff
@@ -0,0 +1,30 @@
+import { CheckBox } from '@yourssu/design-system-react';
+
+import { CategoryContainer } from './Category.style';
+
+interface CategoryProps {
+  id: number;
+  title: string;
+}
+
+const CategoryList: CategoryProps[] = [
+  { id: 1, title: '전체' },
+  { id: 2, title: '교내생활' },
+  { id: 3, title: '취미' },
+  { id: 4, title: '건강' },
+  { id: 5, title: 'IT' },
+  { id: 6, title: '지식' },
+  { id: 7, title: '라이프스타일' },
+  { id: 8, title: '기타' },
+];
+
+export const Category = () => {
+  return (
+    <CategoryContainer>
+      <div>카테고리 유형</div>
+      {CategoryList.map((index) => (
+        <CheckBox key={index.id}>{index.title}</CheckBox>
+      ))}
```

Comment:
```suggestion
{CategoryList.map((category) => (
   <CheckBox key={category.id}>{category.title}</CheckBox>
 ))}
```
개인적으로 `index` 보다는 `category`로 네이밍하는게 더 직관적인 것 같습니당

```suggestion
{CategoryList.map(({id, title}) => (
   <CheckBox key={id}>{title}</CheckBox>
))}
```
아니면 `CategoryList`를 순회하는거니까 디스트럭처링 써도 괜찮을 것 같네요

Reply:
아래 방법 괜찮은 것 같네용

Reply:
저도 2 좋은 거 같아요 ㅎ ㅎ

Reply:
2로 수정하겠습니다ㅎㅎ